src/clients/deep-infra.ts

```diff
@@ -30,15 +31,17 @@ export class DeepInfraClient {
     config?: AxiosRequestConfig,
   ): Promise<AxiosResponse<T>> {
     const headers = {
+      "User-Agent": USER_AGENT,
       "Content-Type": "application/json",
+      ...config?.headers,
```

Comment:
So the reason the headers are expanded here is so that they can override the UA and CT but not the Authorization? Is there a reason for this decision? I mean why not?